src/Workspaces/Core/Portable/Diagnostics/Extensions.cs

```diff
@@ -567,24 +567,4 @@ public static bool IsReportedInDocument(Diagnostic diagnostic, TextDocument targ
 
         return false;
     }
-
-    public static ImmutableArray<DiagnosticAnalyzer> FilterAnalyzers(
-        this ImmutableArray<DiagnosticAnalyzer> analyzers,
-        ImmutableHashSet<string> analyzerIds)
```

Comment:
no need for this anymore (which is really good).  the host and OOP do not talk about analyzers.